src/containers/ListingPage/ListingContainer.tsx

```diff
@@ -125,6 +125,22 @@ const ListingContainer = ({
     }
   };
 
+  useEffect(() => {
+    const handleFilterLanguageChange = (selectedFilter: string): void => {
+      const filterList = Array.from(filters.keys());
+      if (!filterList.includes(selectedFilter)) {
+        handleRemoveFilter();
+      }
+    };
+    if (!data && queryParams.filters[0]) {
+      handleFilterLanguageChange(queryParams.filters[0]);
+    }
+  }, [data, filters, handleRemoveFilter, queryParams.filters]);
+
+  if (!data) {
+    return <></>;
+  }
```

Comment:
Hvorfor er denne lagt inn? Tror det fører til at spinneren ikke vises

Reply:
La til conditional return for å unngå kræsj ved manglende data. Har du tanker om hvordan det heller kunne løses?

Reply:
Kan evt. bytte ut tom div med spinner. (Loading og spinner benyttes i ListingView, men først ved om man ønsker å laste inn mer.)

Reply:
Fungerer å bytte ut tom div med spinner og gjøre den avhengig av loading istedenfor data. Pusher opp ny commit straks

Reply:
Når kræsjer det ved manglende data? Blir ikke bare concepts -> [] og totalCount -> 0, med mindre jeg har oversett noe? Også vises det en spinner i ListingView

Reply:
Det kommer en feil ved at det i `FilterListPhone` (komponent brukt i `ListingView`, fra ndla-ui) har en .map undefined. concepts => [ ] så ikke ut til å løse det. 

Spinneren i ListingView vises ikke før showLoadMore også er true. Så det er ikke en spinner der ved mindre en trykker på "Last mer".

Reply:
Det fungerer det du har nå, men ville likevel skrevet det om siden feilen egentlig ikke har noe med `data` å gjøre. Feilen kommer av at getFilters() returnerer arrays med undefined siden selectedListFilter ikke finnes i filters, så ville heller håndtert det der

Reply:
Godt poeng. Pushet et forslag til løsning nå.